tests/test_xopen.py

```diff
@@ -163,16 +163,16 @@ def lacking_pbzip2_permissions(tmp_path):
         yield
 
 
-@pytest.fixture(params=[1024, 2048, 4096])
-def create_large_file(tmp_path, request):
+@pytest.fixture
+def create_large_file(tmp_path):
     def _create_large_file(extension):
         path = tmp_path / f"large{extension}"
         random_text = "".join(
             random.choice("ABCDEFGHIJKLMNOPQRSTUVWXYZ\n") for _ in range(1024)
         )
         # Make the text a lot bigger in order to ensure that it is larger than the
         # pipe buffer size.
-        random_text *= request.param
+        random_text *= 1024
```

Comment:
The pipe buffer size is now exactly 1 MB since we always use the maximum available pipesize. Can the default be set to 2048 and the comment be slightly adjusted? 
Only processing 2048  and not 1024 and 4096 anymore should already help quite a lot with the speed.

Reply:
Done!